index.html

```diff
@@ -41,24 +45,36 @@
 	//zoomToPolygon() zooms the map to the district extent
 
 			function runWhenLoadComplete() {
-				if (!map.loaded()) {
+				if (!map.loaded() || !map.getLayer('texas-school-districts-poly')) {
 					setTimeout(runWhenLoadComplete, 100);
 				}
 				else {
 					populateZoomControl("school-districts-control", "texas-school-districts", "NAME", "Texas School Districts");
```

Comment:
The `map.moveLayer()` calls after this comment are what sort out the label stacking order.  `'texas-school-districts-poly'` acts as the translucent mask layer to fade everything out, so I needed to put all the pre-K layers behind it.  Then I realised I could put everything behind the labels by first putting the mask layer behind the lowest set of labels in the stacking order, which appears to consistently be `'country-label-sm'`.